roles/kubernetes/node/templates/kubelet-config.v1beta1.yaml.j2

```diff
@@ -160,6 +160,13 @@ seccompDefault: {{ kubelet_seccomp_default | bool }}
 {% if kubelet_cpu_manager_policy is defined %}
 cpuManagerPolicy: {{ kubelet_cpu_manager_policy }}
 {% endif %}
+{% if kubelet_cpu_manager_policy_options is defined %}
+cpuManagerPolicyOptions:
+{% for item in kubelet_cpu_manager_policy_options %}
+  {% set key, value = item.split("=") %}
```

Comment:
It looks a bit weird to take a list of vars and then split it, could you change `kubelet_cpu_manager_policy_options` to a dict instead and directly use that var (with `to_nice_yaml` like some other examples in that file)?

Reply:
Ok, changed it as you suggested. Tested and works as before, thanks!